chart-tests/application/ci/test-role-binding-values.yaml

```diff
@@ -0,0 +1,8 @@
+serviceAccount:
+  rbac:
+    - kind: RoleBinding
```

Comment:
please add a test with multiple bindings of same kind, should fail with the current binding naming approach

Reply:
done

Reply:
test hasn't been updated though.
```diff
  rbac:
    - kind: RoleBinding
      roleType: Role
      roleName: admin
+   - kind: RoleBinding
+     roleType: Role
+     roleName: edit
```

Reply:
Ah, dammit, was the copied file for local tests 😭